diag_manager/fms_diag_yaml_object.F90

```diff
@@ -67,38 +59,38 @@ module fms_diag_yaml_object_mod
      character (len=:), allocatable :: file_timeunit !< The unit of time
      character (len=:), allocatable :: file_unlimdim !< The name of the unlimited dimension
      logical :: file_write
-     character (len=:), allocatable :: string_file_write !< false if the user doesn’t want the file to be 
+     character (len=:), allocatable :: string_file_write !< false if the user doesn’t want the file to be
```

Comment:
shouldn't this be a logical?  (I realize I probably did this)

Reply:
It is read in a string and used to set the logical `file_write` bellow.

Reply:
Got it